modules/storage/main.tf

```diff
@@ -47,7 +47,7 @@ resource "aws_db_instance" "km_db" {
   allocated_storage         = 20
   engine                    = "postgres"
   engine_version            = "10.6"
-  instance_class            = "db.t3.medium"
+  instance_class            = "db.m5.xlarge"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure the standard security best practices configuration is followed for AWS RDS instances**
 
**Vulnerability**
The AWS RDS Instance configuration does not comply with security best practices configuration.
 
**Remediation**
Ensure the following values are set as per the company standard - DB Instance type: db.m5.xlarge, Allocated Storage: 200 G, Max Allocated storage: 500 G, Performance Insights Enabled: True, Enhanced monitoring: True.

source: accurics-